<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>881</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>881</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> 程序员计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
    min-width: 120px; /* 最小宽度 */
    max-width: 120px; /* 最大宽度 */
    min-height: 40px; /* 最小高度 */
    max-height: 40px; /* 最大高度 */
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_string_tool">
     <attribute name="title">
      <string>字符串工具</string>
     </attribute>
     <widget class="QPushButton" name="hex_to_string">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>字符转十六进制</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="output_page_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>651</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="input_page_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="string_to_tab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>十六进制转字符</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_sting_len">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>字符串长度计算</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; 设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="add_0x_prefix">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>50</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>输出加0x前缀</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="add_space">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>80</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>输出字符隔开 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_all">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>全部清空</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="set">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>151</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="get_byte_len">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>字节数组长度计算</string>
      </property>
     </widget>
     <zorder>set</zorder>
     <zorder>hex_to_string</zorder>
     <zorder>output_page_1</zorder>
     <zorder>input_page_1</zorder>
     <zorder>string_to_tab</zorder>
     <zorder>get_sting_len</zorder>
     <zorder>label</zorder>
     <zorder>add_0x_prefix</zorder>
     <zorder>add_space</zorder>
     <zorder>clear_all</zorder>
     <zorder>get_byte_len</zorder>
    </widget>
    <widget class="QWidget" name="tab_conversion">
     <attribute name="title">
      <string>进制转换</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>411</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_instruction">
     <attribute name="title">
      <string>说明</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
